data/json/mapgen/shelter.json

```
@@ -5,16 +5,29 @@
     "subtype": "collection",
     "//": "now used for shelters that have been looted by kids, see SUS groups for general shelter supplies",
     "items": [
-      [ "jacket_evac", 10 ],
-      [ "emer_blanket", 40 ],
       [ "wrapper", 90 ],
+      [ "evac_pamphlet", 80 ],
       [ "glass_shard", 80 ],
+      [ "emer_blanket", 40 ],
       [ "bottle_plastic", 30 ],
       [ "plastic_knife", 20 ],
       [ "plastic_spoon", 20 ],
       [ "plastic_fork", 20 ],
       [ "bowl_plastic", 10 ],
-      [ "evac_pamphlet", 80 ],
+      [ "jacket_evac", 15 ],
+      [ "folding_poncho", 10 ],
+      [ "sponge", 10 ],
+      [ "mask_dust", 9 ],
+      [ "handflare_dead", 9 ],
+      [ "whistle", 7 ],
+      [ "teddy_bear", 7 ],
+      [ "vinegar", 6 ],
```

```suggestion
      { "item": "vitamins", "prob": 6, "charges-min": 1 },
```
That's all you need to do to fix it.